app/services/api/webflow-api.ts

```diff
@@ -31,75 +31,137 @@ import {
   cleanedWorkshops,
   convertScheduleToScheduleCard,
 } from "./webflow-helpers"
+import { queryClient } from "./react-query"
 
-const useWebflowAPI = <T>(key: string, collectionId: string, enabled = true) =>
-  useQuery({
-    queryKey: [key],
-    queryFn: async () => {
-      const { data } = await axiosInstance.get<PaginatedItems<T>>(
-        `/collections/${collectionId}/items`,
-      )
-      return data.items
-    },
-    enabled,
-  })
+const getCollectionById = async <T>(collectionId: string) => {
+  const { data } = await axiosInstance.get<PaginatedItems<T>>(`/collections/${collectionId}/items`)
+  return data.items
+}
+
+const useRecommendationsOptions = {
+  queryKey: [RECOMMENDATIONS.key, RECOMMENDATIONS.collectionId],
```

Comment:
Not sure entirely how necessary this is, but it seems that the docs recommend adding queryKeys for any ids that are associated with a queryFn: https://tanstack.com/query/v4/docs/react/guides/query-keys#if-your-query-function-depends-on-a-variable-include-it-in-your-query-key

I think this is so that we can invalidate specific keys, like if the collectionId ever needed to change for some reason.